src/components/Footer/Footer.scss

```
@@ -1 +1,64 @@
 @import '../../styles/variables.scss';
+
+.Footer {
+  padding: 2rem 0 3rem;
+  background: #000;
+  font-size: 0.9rem;
+
+  .footerWrapper {
+    width: 970px;
+    margin: 0 auto;
+
+    .footerUp {
+      padding-bottom: 2rem;
+      display: flex;
+      justify-content: space-between;
+      border-bottom: 1px solid #999;
+
+      .logoWrapper {
+        h2 {
+          font-family: 'Tangerine', cursive;
+          margin-bottom: 1rem;
+          font-size: 2.4rem;
+          color: white;
+        }
+        p {
+          color: #999;
+          line-height: 1.1rem;
+        }
+      }
```

```suggestion
        h2 {
          font-family: 'Tangerine', cursive;
          margin-bottom: 1rem;
          font-size: 2.4rem;
          color: white;
        }
        
        p {
          color: #999;
          line-height: 1.1rem;
        }
      }
```
디테일이 코드퀄리티를 나타냅니다! 매우 사소하고 간단한 것이라도 지금부터 소윤님만의 깔끔하고 훌륭한 습관을 만들어가보시죠!

---

앞으로도 쭉 습관을 들이도록 하겠습니다! 바로 수정하겠습니다 감사합니다!!!🤭